qr_code/qrcode/constants.py

```diff
@@ -1,17 +1,13 @@
 from datetime import datetime
 
-from qrcode import ERROR_CORRECT_L, ERROR_CORRECT_M, ERROR_CORRECT_Q, ERROR_CORRECT_H
-
-from qr_code.qrcode.image import SVG_FORMAT_NAME
-
 QR_CODE_GENERATION_VERSION_DATE = datetime(year=2019, month=4, day=11, hour=15)
```

Comment:
Since we switched from qrcode to Segno, we must update this date to invalidate any cached result.

Reply:
See changes in 74c4944

I wasn't sure if the date must be more in the past or in the future. Please recheck.